apis/v1alpha4/conversion.go

```diff
@@ -24,7 +24,7 @@ import (
 
 // Convert_v1beta1_VirtualMachineCloneSpec_To_v1alpha4_VirtualMachineCloneSpec is an autogenerated conversion function.
 //
-//nolint:golint,revive,stylecheck
+//nolint:golint
```

Comment:
```suggestion
```

I think we can just drop this too.

Reply:
I would usually expect this stuff to be covered by exclude rules (but I'm not sure which findings we have here)

Reply:
I removed the `//nolint:golint` here and it had no findings.

Reply:
Ah, maybe the nolint finder doesn't find it if we ignore stuff via exludes

Reply:
jep

Reply:
Thanks for this! 👍